idb/postgres/internal/encoding/types.go

```diff
@@ -82,24 +79,26 @@ type signedTxnWithAD struct {
 }
 
 type trimmedAccountData struct {
-	basics.AccountData
-	AuthAddrOverride crypto.Digest `codec:"spend"`
+	baseAccountData
```

Comment:
`baseAccountData` has fewer fields than `basics.AccountData` (i.e. the fields you added below).

Are we going to drop data because of this change?

Reply:
i.e. do we lose participation keys?

Reply:
This is all still very hard to review.

This type and its convert functions are only used for testing, I looked at the master branch and it's always been this way.

Do you know why there is `convertTrimmedAccountData` and `convertTrimmedLcAccountData`?

Reply:
this is when `convertTrimmedLcAccountData` is added, [31eec1](https://github.com/algorand/indexer/commit/31eec101eb41137e45c582a9375201df85caa4ba#diff-6aa11858567176a476547ed480d0ffa95c8c78d2856390385fd101a86bc33546)

Reply:
Sounds like the "unlimited accounts" PR deprecated `trimmedAccountData` everywhere except for in the unit tests. Maybe `insertAccount` should be modified to use `EncodeTrimmed[Lc|Sdk]AccountData`.

Reply:
yes, that's what I'm trying to do with https://github.com/algorand/indexer/pull/1461/files#diff-6aa11858567176a476547ed480d0ffa95c8c78d2856390385fd101a86bc33546R369. I can't use EncodeTrimmedLcAccountData because it missing these fields 
```
MicroAlgos:         uint64(ad.MicroAlgos),
RewardsBase:        ad.RewardsBase,
RewardedMicroAlgos: uint64(ad.RewardedMicroAlgos)
```

Reply:
actually, `insertAccount` doesn't use these fields either.  `EncodeTrimmedLcAccountData` might work.

Reply:
`insertAccount` has been updated to use `EncodeTrimmedLcAccountData`